sstables/sstable_version.cc

```diff
@@ -48,6 +48,8 @@ sstable_version_constants::component_map_t sstable_version_constants::create_com
         { component_type::Scylla, "Scylla.db" },
         { component_type::TemporaryTOC, TEMPORARY_TOC_SUFFIX },
         { component_type::TemporaryStatistics, "Statistics.db.tmp" },
+        { component_type::Rows, "Rows.db" },
+        { component_type::Partitions, "Partitions.db" },
```

Comment:
I'd have expected a change in foreign_sstable_open_info to be in this patch, perhaps it arrives later.

Reply:
It arrives after we actually start opening the files, in a later commit. Here we only add enums.

Reply:
Yes I just saw.